neet/boolean/randomnet.py

```diff
@@ -59,12 +58,16 @@ def random_logic(logic_net, p=0.5, connections='fixed-structure', fix_external=F
             "connections must be 'fixed', 'fixed-in-degree', 'fixed-mean-degree', or 'free'")
 
 
-def _random_binary_states(n, p):
+def random_binary_states(n, p):
     """
     Return a set of binary states. Each state has length `n` and the probability
```

Comment:
We might update the documentation here so that we remember what we’re actually doing (returning np states, or probabilistically between floor(np) and ceil(np) if np is not an integer?)